src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs

```diff
@@ -693,5 +821,34 @@ private void OnCameraTransitionFinished(IntPtr data)
         {
             cameraTransitionFinishedEventHandler?.Invoke(this, EventArgs.Empty);
         }
+
+        // Callback for capture finished signal
+        private void OnCaptureFinished(IntPtr data, int captureId, IntPtr capturedImageUrl)
+        {
+            CaptureFinishedEventArgs e = new CaptureFinishedEventArgs();
+            e.CaptureId = captureId;
+            ImageUrl ret = Registry.GetManagedBaseHandleFromNativePtr(capturedImageUrl) as ImageUrl;
+            if (ret != null)
+            {
+                NUI.Interop.BaseHandle.DeleteBaseHandle(new HandleRef(this, capturedImageUrl));
```

Comment:
Inputed `capturedImageUrl` Don't have ownership of memory.
We should not delete basehandle here.

```suggestion
```